bridges/TwitterBridge.php

```diff
@@ -236,9 +240,66 @@ private function getApiURI() {
 	}
 
 	public function collectData(){
-		$html = '';
-		$page = $this->getURI();
-		$data = json_decode($this->getApiContents($this->getApiURI()));
+		// $data will contain an array of all found tweets (unfiltered)
+		$data = null;
+		// Contains user data (when in by username context)
+		$user = null;
+		// Array of all found tweets
+		$tweets = array();
+
+		// Get authentication information
+		$this->getApiKey();
+
+		// Try to get all tweets
+		switch($this->queriedContext) {
+		case 'By username':
+			$user = $this->makeApiCall('/1.1/users/show.json', array('screen_name' => $this->getInput('u')));
+			if (!$user) {
+				returnServerError('Requested username can\'t be found.');
+			}
+
+			$params = array(
+				'user_id'	 => $user->id,
+				'tweet_mode' => 'extended'
+			);
+
+			$data = $this->makeApiCall('/1.1/statuses/user_timeline.json', $params);
+			break;
+
+		case 'By keyword or hashtag':
+			$params = array(
+				'q'					=> urlencode($this->getInput('q')),
+				'tweet_mode'		=> 'extended',
+				'tweet_search_mode' => 'live',
+			);
+
+			$data = $this->makeApiCall('/1.1/search/tweets.json', $params)->statuses;
+			break;
+
+		case 'By list':
+			$params = array(
+				'slug'				=> strtolower($this->getInput('list')),
+				'owner_screen_name' => strtolower($this->getInput('user')),
+				'tweet_mode'		=> 'extended',
```

Comment:
I suggest to use spaces between "=>" operator.

Reply:
Okay. I will convert them later